src/main/java/com/redhat/devtools/lsp4ij/console/actions/ApplyLanguageServerSettingsAction.java

```diff
@@ -41,11 +51,55 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
     @Override
     public void update(@NotNull AnActionEvent e) {
-        e.getPresentation().setEnabled(languageServerView.isModified());
+        boolean modified = languageServerView.isModified();
+        boolean isShown = languageServerView.isSaveTipShown();
+
+        // Only show the tooltip once per configuration change
+        if (modified && !isShown) {
+            showBalloon();
+        }
+        // If configuration is returned to match the unmodified state, reset the tooltip to be shown again
+        if (!modified) {
+            languageServerView.isSaveTipShown(false);
+        }
+
+        e.getPresentation().setEnabled(modified);
     }
 
     @Override
     public @NotNull ActionUpdateThread getActionUpdateThread() {
         return ActionUpdateThread.BGT;
     }
+
+    private void showBalloon() {
+        languageServerView.isSaveTipShown(true);
+        JBLabel jbPanel = new JBLabel(LanguageServerBundle.message("action.lsp.detail.apply.balloon"));
+
+        BalloonBuilder builder = JBPopupFactory.getInstance()
+                .createBalloonBuilder(jbPanel)
+                .setFadeoutTime(800)

```

Comment:
Could you add please a comment which explains how many times the balloon is shown (800ms ?)